cmd/algorand-indexer/daemon.go

```diff
@@ -31,31 +28,17 @@ var (
 	developerMode    bool
 	tokenString      string
 
-	configFilePath string
-
 	logger *log.Logger
 )
 
-func init() {
-	logger = log.New()
-	logger.SetFormatter(&log.JSONFormatter{})
-	logger.SetOutput(os.Stdout)
-	logger.SetLevel(log.InfoLevel)
-}
-
 var daemonCmd = &cobra.Command{
 	Use:   "daemon",
 	Short: "run indexer daemon",
 	Long:  "run indexer daemon. Serve api on HTTP.",
 	//Args:
 	Run: func(cmd *cobra.Command, args []string) {
-		if configFilePath != "" {
-			cf, err := os.Open(configFilePath)
-			maybeFail(err, "%s: %v", configFilePath, err)
-			err = configFromStream(cf)
-			maybeFail(err, "%s: %v", configFilePath, err)
-			cf.Close()
-		}
+		util.BindFlags(cmd, "INDEXER")
```

Comment:
Let's make it a constant

Reply:
util.BindFlags() is never called any other way, the string can just live inside that function

Reply:
I don't understand what it has to do with being called by another function. Strings, especially if they are used more than once, need to be constants. It makes it easier to read and to change.

Reply:
I'm saying we could change all the calls to `util.BindFlags(cmd)` and "INDEXER" only exists inside the function and passes the don't-repeat-yourself principle, it can't get out of sync in different places.

Reply:
Thanks, great suggestions. It should totally be a constant and I got a little ahead of myself when making the BindFlags function.